DEV-2/DEV-2.Tests/TranscriptionMakerTests.cs

```diff
@@ -0,0 +1,83 @@
+﻿
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace DEV_2.Tests
+{
+    [TestClass]
+    public class TranscriptionMakerTests
+    {
+        /// <summary>
+        /// Checks if ReplaceUnstressedO method replaces unstressed 'o' with 'a'.
+        /// </summary>
+        /// <param name="input"></param>
+        /// <param name="expected"></param>
+        [TestMethod]
+        [DataRow("молоко+", "малако")]
+        [DataRow("полено+", "палено")]
+        [DataRow("о+", "о")]
+        [DataRow("о", "а")]
+        public void ReplaceUnstressedOTest(string input, string expected)
+        {
+            var converter = new LetterObjectConverter();
+            var transcriptionMaker = new TranscriptionMaker(converter.ConverStringToLetterObjectList(input));
+
+            transcriptionMaker.ReplaceUnstressedO();
+
+            Assert.AreEqual(expected, converter.DisplayTranscription(transcriptionMaker.incertedString).ToString());
```

Comment:
move to variable 'converter.DisplayTranscription(transcriptionMaker.incertedString).ToString()'. It will be easier to read